### PR TITLE
[Proposal] init run dev server

### PR DESCRIPTION
**Pull request by dbrrt**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

N/A

This PR aims to introduce a dev server for `run` function, and some isolation ideas between main, index and run logics.

--

Thanks for contributing!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "main": "src/index.js",
   "type": "module",
   "scripts": {
+    "dev": "nodemon ./src/dev.js localhost",
```
Comment (tripodsan) [inline package.json:9]: standart is:

```suggestion
    "start": "nodemon ./src/dev.js localhost",
```

so you can just use `npm start`

**Review by tripodsan — CHANGES_REQUESTED** [review]
that's too complicated, just use the https://github.com/adobe/helix-deploy/blob/main/src/DevelopmentServer.js

the DevelopmentServer mimics a function request and you don't need to dissect your code.

example:

https://github.com/adobe/helix-admin/blob/main/test/dev/server.mjs

File: package.json (hunk 1 above)
Comment (tripodsan) [reply package.json:9]: also, better put the nodemon config in it's own structure:

see https://github.com/adobe/helix-admin/blob/102a7c24bb57691f71c7cde9a26eb48457aa67e0/package.json#L134-L141

Comment (dbrrt) [reply package.json:9]: @tripodsan Got it, Thanks, will revert to leverage the main. I'll also integrate the franklin deploy dev server.

Can you give me access to `adobe/helix-admin` repo so I can compare and adjust with the preferred nodemon structure?

**Pull request closed without merging**